tests/playwright/Report.md

```diff
@@ -0,0 +1,176 @@
+- Selenium
+- Puppeteer
+- Cypress
+- Playwright(maybe the new kid on the block but it's gaining quite a trajectory among Software developers and testers)
+
+### Project Context
+The Playwright evaluation is a result of a need for potential replacement of [WebdriverIO](https://webdriver.io/) with one of the best non-selenium modern test automation framwork. Wikimedia supports a large number of repositories. So, in order to ensure good code practices across all these repositories, an extensive amount of testing is performed and one of the tests performed is an end to end test. WebdriverIO is the current browser automation framework being used for implementing the end-to-end tests. However, with the recent increase in awareness about end-to-end testing, a number of equally competitive non-selenium solutions have been introduced and one of them is [Microsoft Playwright](https://github.com/microsoft/playwright).
+
+At the time of choosing which automation framework to use in the implementation of the test automation, extensive research was done and an [evaluation of WebdriverIO](https://filipin.eu/selenium-javascript) was carried out by Željko Filipin, who is a Senior Engineer in Test at Wikimedia Foundations and also one of my mentors. In search for a potential replacement of WebdriverIO, a number of modern and non-selenium automation frameworks have been evaluated against WebdriverIO. These evaluations were done by Soham Parekh who was Google Summer of Code 2020 intern and, is also one of my mentors and Sim Tran(correct name to be verified). These evaluations can be found in the links below;
+- [Cypress Evaluation](https://www.mediawiki.org/wiki/User:SimTran/Cypress_vs_WebdriverIO:_What_are_the_differences%3F)
+- [Puppeteer Evaluation](https://www.sohamp.dev/blog/2020-08-24-wmf-gsoc2020/)
+
+
+So, this brings us to the current framework being evaluated against WebdriverIO, which is [Microsoft Playwright](https://github.com/microsoft/playwright), also known as Playwright. The evaluations progress involved the following steps;
+
+- Configuring and Setting up of Wikimedia-core to run on CI, in this case on Github actions.
+- Implementation of all mediawiki-core tests in Playwright
+- Runing mediawiki-core tests in WebdriverIO
+- Analysis of both WebdriverIO and Playwright tests in terms of speed and stability
+
+### Playwright Vs WebdriverIO
+
+[WebdriverIO](https://github.com/webdriverio/webdriverio) is a test automation framework that allows you to run tests based on the Webdriver protocol and Appium automation technology. WebdriverIO is known to be;
```

Comment:
```suggestion
[WebdriverIO](https://github.com/webdriverio/webdriverio) is a test automation framework that allows you to run tests based on the WebDriver protocol and Appium automation technology. WebdriverIO is known to be;
```

WebDriver, according to https://www.selenium.dev/documentation/en/webdriver/

Reply:
Can you please elaborate on this?